### ISSUE TITLE
'Landing Page'

### ISSUE THREAD

**Issue by Docfips** [lumen-notes/lumen#458]
Is there a possibility to be able to have a specific page upon loading? Not a req or anything, just curious if it's possible.

**Comment by colebemis** [comment]
Just to clarify, do you mean choosing a specific note to open by default when you go to app.uselumen.com?

**Comment by Docfips** [comment]
Yessir!

For instance, if I made some sort of a dashboard page with tag links, I'd go to the site and instead of the full notes, I could open that specific 'dashboard'

**Comment by colebemis** [comment]
I see. That makes sense. That's not something I'd thought about before. The closest we have to that would be to pin your "dashboard" note so it always shows up first

**Comment by Docfips** [comment]
Correct. At the moment it's just a pin. Just curious *if* it's possible. Like I said, not actually a big thing, was just curious.
